tree/tree/inc/TBranch.h

```diff
@@ -224,7 +225,7 @@ class TBranch : public TNamed , public TAttFill {
 
    static  void      ResetCount();
 
-   ClassDef(TBranch,12);  //Branch descriptor
+   ClassDef(TBranch, 13); // Branch descriptor
```

Comment:
No need to update the version number unless the on-file format changed (i.e. if one of the data member changed)